modules/serverless-ossm-setup.adoc

```diff
@@ -19,14 +19,20 @@ ifdef::openshift-dedicated,openshift-rosa[]
 endif::[]
 
 * You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
-* Install {SMProductName}. {ServerlessProductName} with {SMProductShortName} only is supported for use with {SMProductName} version 2.0.5 or later.
+
+* Install the {SMProductName} Operator and create a `ServiceMeshControlPlane` resource in the `istio-system` namespace. If you want to use mTLS functionality, you must also set the `spec.security.dataPlane.mtls` field for the `ServiceMeshControlPlane` resource to `true`.
```

Comment:
I noticed that we change tenses/voices in the prerequisites from "You have created..." to "Install {SMProductName}". The SSG does state "Use parallel language when you write prerequisites. For example, if one bullet is a complete sentence, write the other bullets as complete sentences. But one bullet can be passive voice and another active voice." 

If you don't think switching voices is a problem, then LGTM.

Reply:
I think for now it's OK because the bullet was already in this tense, and changing it to "you have installed" might make the second sentence seem weird, i.e. "if you want to use..." because it would be changing tenses mid-bullet I think.